### DISCUSSION TITLE
make `M t` appear in the commands list when rebasing with `-r` (--rebase-merges)

### DISCUSSION THREAD

**Discussion by danielhvs** [magit/magit#5201] [Ideas]
When rebasing including merge commits, would be nice for the command `M t`: `git-rebase-merge-toggle-editmsg` to be shown together with the helper comments:

```bash
# p        pick = use commit
# r        reword = use commit, but edit the commit message
# e        edit = use commit, but stop for amending
(...)
# C-c C-c  tell Git to make it happen
# C-c C-k  tell Git that you changed your mind, i.e. abort
(...) 
``

**Comment by tarsius** [comment]
I've just looked into doing this (probably once more) and have decided to not do it. The problem is that this command would have to be handled differently from all other commands. It's not one of those inserted by Git, nor one of those that belongs in the block of commands we inject at the beginning, nor a fixup commit. A new special case would be need to insert it next to `git-rebase-merge` and that is not worth the effort.

**Reply by danielhvs** [reply]
> I've just looked into doing this (probably once more) and have decided to not do it. The problem is that this command would have to be handled differently from all other commands. It's not one of those inserted by Git, nor one of those that belongs in the block of commands we inject at the beginning, nor a fixup commit. A new special case would be need to insert it next to `git-rebase-merge` and that is not worth the effort.

No problem! Thank you very much for taking the time @tarsius
